modules/_labs/drawer/react/lib/DrawerHeader.tsx

```diff
@@ -74,7 +74,7 @@ const CloseButton = styled(IconButton)({
 export default class DrawerHeader extends React.Component<DrawerHeaderProps, {}> {
   public render() {
     const {
-      closeIconLabel = 'Close',
+      'aria-label': closeIconAriaLabel = 'Close',
```

Comment:
thoughts on this approach? not my favorite, but we week the attribute the same name, and just add the type as `closeIconAriaLabel` for clarification?

Reply:
Hmm. So props spread on the `HeaderContainer` element except for `title`, `id`, and now `aria-label`. I'm not sure it is obvious that `aria-label` means "`aria-label` of the close icon".

I think `closeIconAriaLabel` is more descriptive of purpose than `aria-label`.

Reply:
I think if the element receiving the aria label is the spread element, I think it is better for us to simply shadow the prop so that our doc system calls it out.

Example:
```tsx
// Make sense to keep `aria-label`
const MyComponent = ({
  'aria-label': ariaLabel = 'Some Default',
  ...elemProps,
}) => {
  return (
    <div aria-label={ariaLabel} {...elemProps}>Something</div>
  )
}

// Maybe it doesn't make sense
const MyComponent = ({
  'aria-label': someAriaLabel = 'Close',
  ...elemProps,
}) => {
  return (
    <div {...elemProps}>
      <SomeOtherComponent aria-label={someAriaLabel} />
  </div>
  )
}
```